### PR TITLE
Create app

### PR DESCRIPTION
**Pull request by sovetnik**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  def application do
+    [
+      extra_applications: [:logger]
+    ]
+  end
+
+  # Run "mix help deps" to learn about dependencies.
+  defp deps do
+    [
+      {:credo, "~> 1.6", only: [:dev, :test], runtime: false}
```
Comment (am-kantox) [inline mix.exs:24]: I am not sure if `credo` makes any  sense in `:test` environment. Also, this is a premature addition, let’s add code when we understand whys and whats.

File: mix.exs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      elixir: "~> 1.12",
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: [:logger]
```
Comment (am-kantox) [inline mix.exs]: There is no single call to [`Logger`](https://hexdocs.pm/logger/Logger.html) in the code so I am unsure why would we start its application here.

File: lib/hello_elixir.ex [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  Hello world.
+
+  ## Examples
+
+      iex> HelloElixir.hello()
+      "heil elixir!"
+
+  """
+  def hello do
+    "heil elixir!"
```
Comment (am-kantox) [inline lib/hello_elixir.ex]: In der deutschen Sprache werden Substantive mit einem Großbuchstaben begonnen.

**Review by am-kantox — APPROVED** [review]
Approved with comments.

File: mix.exs (hunk 2 above)
Comment (sovetnik) [reply mix.exs]: Comented it

File: lib/hello_elixir.ex (hunk 3 above)
Comment (sovetnik) [reply lib/hello_elixir.ex]: Fixed grammar

**Pull request merged**